### PR TITLE
Lint roller

### PR DESCRIPTION
**Pull request by rgbkrk**
Applied the same changes that @betatim applied in the display area.

While I was at it, tried out [enzyme](https://github.com/airbnb/enzyme/) per @kenwheeler.

Not totally happy with how we test our LaTeX display area, though KaTeX does all the heavy lifting.

### CODE REVIEW

File: src/components/html-display.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function HTMLDisplay(props) {
+  return (
+    <div
+      dangerouslySetInnerHTML={{ // eslint-disable-line
```
Comment (betatim) [inline src/components/html-display.js]: hashtag-noqa :grinning:

Comment (rgbkrk) [inline src/components/html-display.js]: :smile:

**Comment by betatim** [comment]
Travis is happy -> merging. Nice work!

**Comment by rgbkrk** [comment]
I can go ahead and make a major release or start tackling #9 first.

**Pull request merged**